docs/Contributors_Guide/github_workflow.rst

```diff
@@ -22,6 +22,269 @@ develop branch.  When a release candidate for METplus has been determined,
+        Under **More options** select a **Start on** date and set the default **Duration** as 6 weeks.
+        Click the **Add** and **Add iteration** buttons to create 5 cycles, each with the default duration
+        of 6 weeks. Modify the cycle names to be **Beta1**, **Beta2**, and so on. Click **Save changes**.
+
+      - Select **Estimate** and click the three dots to the right and **Delete field**.
+
+   - Click the back arrow to return to the project page.
+
+3. Update the project **Workflows**.
+
+   - Click on the three dots to the right of the project name to see **More options**
+     and select **Workflows**.  Modify these settings as follows.
+
+      - Enable the **Item added to project** workflow and set the status to **Backlog**.
+
+      - Enable the **Item reopened** workflow and set the status to **In progress**.
+
+      - Enable the **Item closed** workflow and set the status to **Done**.
+
+      - Enable the **Pull request merged** workflow and set the status to **Done**.
+
+      - Leave all other workflows disabled.
+
+   - Click the back arrow to return to the project page.
+
+4. Create project **Views**.
+
+   - Each view appears as a tab on the project page.  Create a new view as described below.
+
+      - Select the **+ New view** option.
+
+      - Click on the view name to modify it, rename it as **All Cycles**, and hit **Enter**.
+
+      - Click on the down arrow and, under **Configuration**, select the **Fields** option. Enable
+        the options for **Title**, **Repository**, **Assignees**, **Cycle**, **Status**,
+        **Linked Pull Request**, and **Reviewers**. Drag and drop the items to reorder them as listed above.
+
+      - In the resulting view, click the three dots in the **Status** column. Select the **Sort descending**
+        and **Group by values** options.
+
+      - The blue dot on the down arrow for this tab indicates that there are unsaved changes.
+        Select the **Save changes** option.
+
+   - Click on the down arrow and select **Duplicate View**.  Do this 7 times and name/refine these views as follows.
+
+      - View name **All Required** shows all items labelled as *REQUIRED* for the development or official release.
+        Click on the 3 horizontal bars and define the filtering criteria as
+        **label:"required: FOR OFFICIAL RELEASE","required: FOR DEVELOPMENT RELEASE" is:open**.  Click **Save Changes**.
+
+      - View name **Beta1** to **Beta5** shows items for each individual development cycle.
+        Click on the 3 horizontal bars and define the filtering criteria as **cycle:Beta1**, **cycle:Beta2**,
+        and so on.  Click **Save Changes**.
+
+      - View name **Closed Issues** shows issues that have been closed across all development cycles.
+        Click on the 3 horizontal bars and define the filtering criteria as **is:closed is:issue**.
+        Click on the 3 dots in the **Cycle** column and select **Group by values**.  Click **Save Changes**.
+
```

Comment:
The METplus project boards have included another view that I find useful. This isn't mandatory to add.

```suggestion

      - View name **High/Blocker Not Required** shows all items labelled as *HIGH* or *BLOCKER* priority but
        not marked as required for the development or official release.
        Click on the 3 horizontal bars and define the filtering criteria as
        **is:open label:"priority: high","priority: blocker" -label:"required: FOR DEVELOPMENT RELEASE" -label:"required: FOR OFFICIAL RELEASE"**.
        Click **Save Changes**.

```